orca/internal/internal.go

```diff
@@ -32,3 +41,30 @@ var AllowAnyMinReportingInterval interface{} // func(*ServiceOptions)
 //
 // For testing purposes only.
 var DefaultBackoffFunc = ibackoff.DefaultExponential.Backoff
+
+// TrailerMetadataKey is the key in which the per-call backend metrics are
+// transmitted.
+const TrailerMetadataKey = "endpoint-load-metrics-bin"
+
+// ToLoadReport unmarshals a binary encoded [ORCA LoadReport] protobuf message
+// from md and returns the corresponding struct. The load report is expected to
+// be stored as the value for key "endpoint-load-metrics-bin".
+//
+// If no load report was found in the provided metadata, if it cannot be
+// parsed, or if multiple are found, an error is returned.
```

Comment:
Nit: what is "it". Switch to "If no load report was found in the provided metadata, if multiple load reports are found, or if the load report found cannot be parsed, an error is returned."

Reply:
Done.